controllers/provider-aws/charts/provider-aws/templates/configmap-logging.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "name" . }}-logging-config
+  namespace: garden
+  labels:
+    extensions.gardener.cloud/configuration: logging
+data:
+  filter-kubernetes.conf: |-
+    [FILTER]
+        Name                parser
+        Match               kubernetes.cloud-controller-manager*aws-cloud-controller-manager*
+        Key_Name            log
+        Parser              kubeapiserverParser
+        Reserve_Data        True
+
+    [FILTER]
+        Name                parser
+        Match               kubernetes.machine-controller-manager*aws-machine-controller-manager*
+        Key_Name            log
+        Parser              kubeapiserverParser
+        Reserve_Data        True
+
+    [FILTER]
+        Name record_modifier
+        Match *aws-cloud-controller-manager*
+        Record type user
+
+{{/*TODO: should this be put in each extension provider?    */}}
```

Comment:
Nope, AWS is the only control plane exposure controller that implements anything. All other providers deploy only components into the seed that are configured for the shoot provider (not the seed provider).